database/load_questions_answes.sql

```diff
@@ -0,0 +1,21 @@
+USE `labby.db`;
```

Comment:
Nice! Sorry, one last thing I think this file is spelled wrong, maybe we can correct the `answes` :)

Reply:
oh no so sorry!